tests/lib/rules/no-incorrect-calls-with-inline-anonymous-functions.js

```diff
@@ -0,0 +1,258 @@
+      run.schedule('afterRender', this.methodToInvoke);
+      run.schedule('afterRender', methodToInvoke);
+
+      // schedule always allows inline functions
+      run.schedule('afterRender', this, function() {});
+      run.schedule('afterRender', function() {});
+      run.schedule('afterRender', () => {});
+
+      someOtherMethod('foo', this, function() {});
+      someOtherMethod('foo', function() {});
+      someOtherMethod(function() {});
+      someOtherMethod(() => {});
+    `,
+    `
+      import { somethingElse } from '@ember/runloop';
+      scheduleOnce('afterRender', this, function() {});
+      scheduleOnce('afterRender', function() {});
+      once(this, function() {});
+      once(function() {});
+      debounce(this, function() {}, 300);
+      debounce(function() {}, 300);
+
+      run.scheduleOnce('afterRender', this, function() {});
+      run.scheduleOnce('afterRender', function() {});
+      run.once(this, function() {});
+      run.once(function() {});
+
+      scheduleOnce('afterRender', this, () => {});
+      scheduleOnce('afterRender', () => {});
+      once(this, () => {});
+      once(() => {});
+      debounce(this, () => {}, 300);
+      debounce(() => {}, 300);
+      run.scheduleOnce('afterRender', this, () => {});
+      run.scheduleOnce('afterRender', () => {});
+      run.once(this, () => {});
+      run.once(() => {});
+    `,
+    `
+      import { run } from 'not-ember';
+
+      run.once(this, function() {});
+      run.scheduleOnce('afterRender', this, function() {});
+      run.once(function() {});
+      run.scheduleOnce('afterRender', function() {});
+      run.debounce(this, function() {}, 300);
+
+      run.once(this, () => {});
+      run.scheduleOnce('afterRender', this, () => {});
+      run.once(() => {});
+      run.scheduleOnce('afterRender', () => {});
+      run.debounce(this, () => {}, 300);
+    `,
+    `
+      import { run } from '@ember/runloop';
+
+      run.once.somethingElse(this, function() {});
+      run.scheduleOnce.somethingElse('afterRender', this, function() {});
+      run.debounce.somethingElse(this, function() {}, 300);
+      somethingElse.run.once(this, function() {});
+      somethingElse.run.scheduleOnce('afterRender', this, function() {});
+      somethingElse.run.debounce(this, function() {}, 300);
+
+      run.once.somethingElse(this, () => {});
+      run.scheduleOnce.somethingElse('afterRender', this, () => {});
+      run.debounce.somethingElse(this, () => {}, 300);
+      somethingElse.run.once(this, () => {});
+      somethingElse.run.scheduleOnce('afterRender', this, () => {});
+      somethingElse.run.debounce(this, () => {}, 300);
+    `,
+    "someOtherMethod('foo', this, function() {})",
+    "someOtherMethod('foo', function() {})",
+    'someOtherMethod(function() {})',
+    "import OtherThing from 'other-thing';",
+    "import YetOtherThing, { namedThing, anotherNamedThing } from 'yet-other-thing';",
```

Comment:
when I ran the rule on dashboard, at first I got errors because these `import` styles weren't accounted for in the `namedImports` calculation